mmv1/third_party/terraform/data_sources/data_source_google_cloudfunctions2_function.go

```diff
@@ -0,0 +1,57 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type cloudFunction2Id struct {
+	Project  string
+	Location string
+	Name     string
+}
+
+func (s *cloudFunction2Id) cloudFunction2Id() string {
```

Comment:
Given this function isn't reused anywhere, can we inline it? That eliminates the need for a `cloudFunction2Id` type, too.

Reply:
Ok, I will make it inline and eliminate the `cloudFunction2Id` struct.